packages/normalize/css/index.css

```diff
@@ -17,6 +17,55 @@
   font-style: italic;
 }
 
+@font-face {
```

Comment:
Adds 1Kb to built stylesheet.

Reply:
@jaketrent What about the weight added from the font files? The monospace font is pretty seldom used, is there a different way this could be done where it's pulled in when needed rather than adding it to normalize where it will request the monospace fonts whether they're used or not?

Reply:
Hehe, yeah, I verified that. And I've wondered about it several times, so I wrote down what I learned: https://jaketrent.com/post/when-font-face-is-downloaded

Yes, the font file weight will be pulled in when needed.

Reply:
Nice!